pkg/resources/resources_manager_test.go

```diff
@@ -485,4 +486,44 @@ var _ = Describe("ResourcesManger", func() {
 			fakeResourceServer.AssertExpectations(testCallsAssertionReporter)
 		})
 	})
+	Context("PeriodicUpdate", func() {
+		It("Update resources for resource manager", func() {
```

Comment:
can you also add a test when rm.PeriodicUpdateInterval is zero that no updates occur ?

Reply:
added